torch/_inductor/decomposition.py

```diff
@@ -19,15 +19,29 @@
 
 inductor_decompositions = get_decompositions(
     [
+        aten._adaptive_avg_pool2d_backward,
```

Comment:
We need to check that these produce good inductor generated code before adding them here

Reply:
How do I do that? These were originally in the core aten decomp table so I would expect this diff does a noop for inductor.

Reply:
ah, if they were already being added nevermind